airbyte-integrations/connectors/source-amplitude/source_amplitude/manifest.yaml

```diff
@@ -0,0 +1,171 @@
+
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+
+  date_stream_slicer:
+    type: DatetimeStreamSlicer
+    start_datetime:
+      datetime: "{{ format_datetime(config['start_date'], '%Y-%m-%d') }}"
+    end_datetime:
+      datetime: "{{ now_utc().strftime('%Y-%m-%d') }}"
+    datetime_format: "%Y-%m-%d"
+    cursor_granularity: P1D
+    step: P15D
+    cursor_field: "{{ options['stream_cursor_field'] }}"
+
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+
+  annotations_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "annotations"
+      primary_key: "id"
+      path: "/2/annotations"
+      data_field: "data"
+
+  cohorts_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "cohorts"
+      primary_key: "id"
+      path: "/3/cohorts"
+      data_field: "cohorts"
+
+  base_incremental_stream:
+    checkpoint_interval: 10
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      requester:
+        $ref: "*ref(definitions.requester)"
+        request_options_provider:
+          request_parameters:
+            start: "{{format_datetime(stream_slice.start_time, '%Y%m%d') }}"
+            end: "{{format_datetime(stream_slice.end_time, '%Y%m%d') }}"
+      stream_slicer:
+        $ref: "*ref(definitions.date_stream_slicer)"
+
+  average_session_length_stream:
+    $ref: "*ref(definitions.base_incremental_stream)"
+    retriever:
+      $ref: "*ref(definitions.base_incremental_stream.retriever)"
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: CustomRecordExtractor
+          field_pointer: [ "data" ]
+          class_name: source_amplitude.components.AverageSessionLengthRecordExtractor
+    $options:
+      name: "average_session_length"
+      primary_key: "date"
+      path: "/2/sessions/average"
+      stream_cursor_field: "date"
+
+  active_users_stream:
+    $ref: "*ref(definitions.base_incremental_stream)"
+    retriever:
+      $ref: "*ref(definitions.base_incremental_stream.retriever)"
+      requester:
+        $ref: "*ref(definitions.base_incremental_stream.retriever.requester)"
+        request_options_provider:
+          request_parameters:
+            start: "{{format_datetime(stream_slice.start_time, '%Y%m%d') }}"
+            end: "{{format_datetime(stream_slice.end_time, '%Y%m%d') }}"
+            m: "active"
+            i: "1"
+      stream_slicer:
+        $ref: "*ref(definitions.date_stream_slicer)"
+        step: P1M
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: CustomRecordExtractor
+          field_pointer: [ "data" ]
+          class_name: source_amplitude.components.ActiveUsersRecordExtractor
+    $options:
+      name: "active_users"
+      primary_key: "date"
+      path: "/2/users"
```

Comment:
I'm a little bit confused here. It seems like the [/2/users API ](https://www.docs.developers.amplitude.com/analytics/apis/dashboard-rest-api/#get-active-and-new-user-counts) returns something like:
```
{
    "data": {
        "series": [ 
            [46109, 47542],
            [42845, 42626]
        ],
        "seriesMeta": ["United States", "Canada"],
        "xValues": ["2017-08-14", "2017-08-15"]
    }
}
```

However in this manifest, it uses on the `ActiveUsersRecordExtractor` extractor [which expect `seriesLabel`](https://github.com/airbytehq/airbyte/pull/22362/files#diff-ddf43f76d82f3374fff8f08974773811a594ed89a753f52d2462988fd5e622a1R44) and I don't see `seriesLabel` in the template response from Amplitude. Is this expected?

Reply:
Yes, this is expected:
![image](https://user-images.githubusercontent.com/36314070/216643111-cb994afb-272b-40f8-8312-f0dc8da908bc.png)